admin/proc-rbac-ui-create-role.adoc

```diff
@@ -5,7 +5,7 @@ You can create a role in the {product} using the Web UI.
 
 .Prerequisites
 * You have an administrator role in the {product-short}.
-* You have installed the `@janus-idp/backstage-plugin-rbac` plugin in {product-short}. For more information, see xref:rhdh-installing-dynamic-plugins[].
+* You have installed the `@janus-idp/backstage-plugin-rbac` plugin in {product-short}. For more information, see link:{LinkPluginsGuide}[{NameOfPluginsGuide}].
```

Comment:
All links to the docs 404 in the preview, and even when I c&p the links from the attributes doc into a browser, but the ones that are currently published in prod do work. So I assume that this one also works in prod.